frontend/test/metabase/lib/dom.unit.spec.js

```diff
@@ -21,15 +18,19 @@ describe("getSelectionPosition/setSelectionPosition", () => {
     const position = getSelectionPosition(input);
     expect(position).toEqual([3, 6]);
   });
-  it("should get/set selection on contenteditable correctly", () => {
+
+  // NOTE (based on original Tom Robinson's note from 1/7/2019):
+  // this worked in karma but jsdom doesn't have the required APIs on div/contenteditable
+  xit("should get/set selection on contenteditable correctly", () => {
```

Comment:
Maybe these tests can be deleted rather than skipped? Our Cypress test of custom expressions _somewhat_ covers this.

In general, I'd say this unit-level coverage is good, but given the browser APIs required, I think relying on Cypress is fine.

Reply:
Afaic, even better :)